web-portal/components/ExistingEventDetectionAlert.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div class="existing-event-detection-alert" v-show="isShown">
```

Comment:
`v-if` instead of `v-show` would suppress rendering altogether (which is generally my preference in a case like this but it's just a preference).

Reply:
sweet that's what I wanted, but didn't understand how to specify in vue, will change

Reply:
1aad33ca29cb31e60a0b40ab558690885e142d78